app/liquid/views/partials/_form.liquid

```diff
@@ -16,7 +16,8 @@
           <input type="tel" name="{{field.name}}" class="form-control form__content" id="" placeholder="{{ field.label }}" />
         {% when 'checkbox' %}
           <label class="checkbox-label">
-            <input type="checkbox" name="{{field.name}}" class="form__content" /> {{ field.label }}
+            <input name="{{field.name}}" type="hidden" value="0" />
```

Comment:
ohh I remember this, this is what the form helper does. and it works properly in the click test